node/runtime/src/lib.rs

```diff
@@ -491,6 +497,14 @@ impl From<CommitteeMember<CrossChainPublic, SessionKeys>> for BlockAuthor {
 		}
 	}
 }
+impl From<BlockAuthor> for Option<MainchainKeyHash> {
+	fn from(value: BlockAuthor) -> Self {
+		match value {
+			BlockAuthor::Incentivized(_, key) => Some(key.hash()),
+			_ => None,
```

Comment:
if there is only one more element, let's make it without wildcard, in case someone will add new element in the future, we will have an compilation error
```suggestion
			BlockAuthor::ProBono(_) => None,
```

Reply:
yeah